lxd/storage/drivers/generic_vfs.go

```diff
@@ -556,6 +556,18 @@ func genericVFSBackupVolume(d Driver, vol Volume, tarWriter *instancewriter.Inst
 				}
 
 				d.Logger().Debug(logMsg, log.Ctx{"sourcePath": mountPath, "prefix": prefix})
+
+				// Follow the target if mountPath is a symlink.
+				// Functions like filepath.Walk() won't list any directory content otherwise.
+				target, err := os.Readlink(mountPath)
```

Comment:
@monstermunchkin for consistency, should we add this same logic to the bit above where it has `filepath.Walk` around where it says "Copying custom config volume"?

Reply:
Nope. That code is only executed `if v.IsVMBlock()`. The `"Copying custom config volume"` log message will actually never show up and could be removed safely.

Reply:
True, although won't this become relevant again once you integrate it with https://github.com/lxc/lxd/pull/9300?

Reply:
No, it won't. The `if v.IsVMBlock()` is placed around the VM config volume in #9300.

Reply:
OK great thanks.